### ISSUE TITLE
Get folder remappings from Solidity config file

### ISSUE THREAD

**Issue by naddison36** [naddison36/sol2uml#87]
This is for filesystem source code and from Etherscan

An example contract on Etherscan with remappings is the Replica contract from Nomad https://etherscan.io/address/0xB92336759618F55bd0F8313bd843604592E27bd8

```json
{
  "remappings": [
    "@ensdomains/=node_modules/@ensdomains/",
    "@nomad-xyz/=packages/",
    "@openzeppelin/=node_modules/@openzeppelin/",
    "@summa-tx/=node_modules/@summa-tx/",
    "ds-test/=lib/forge-std/lib/ds-test/src/",
    "eth-gas-reporter/=node_modules/eth-gas-reporter/",
    "forge-std/=lib/forge-std/src/",
    "hardhat/=node_modules/hardhat/",
    "prettier-plugin-solidity/=node_modules/prettier-plugin-solidity/",
    "solscripts/=packages/upgrade/solscripts/",
    "test/=packages/upgrade/solscripts/test/",
    "solscripts/=packages/upgrade/solscripts/",
    "test/=packages/upgrade/solscripts/test/"
  ],
  "optimizer": {
    "enabled": true,
    "runs": 999999
  },
  "metadata": {
    "bytecodeHash": "none"
  },
  "outputSelection": {
    "*": {
      "*": [
        "evm.bytecode",
        "evm.deployedBytecode",
        "devdoc",
        "userdoc",
        "metadata",
        "abi"
      ]
    }
  },
  "evmVersion": "istanbul",
  "libraries": {}
}

**Comment by Girlsavenue2** [comment]
#123 #122 #42 #121 #74

**Comment by naddison36** [comment]
This was also an issue for Uniswap `UniversalRouter` at [0xEf1c6E67703c7BD7107eed8303Fbe6EC2554BF6B](https://etherscan.io/address/0xEf1c6E67703c7BD7107eed8303Fbe6EC2554BF6B#code)

`sol2uml storage -d 0xEf1c6E67703c7BD7107eed8303Fbe6EC2554BF6B -o ./testDiagrams/storage/`

**Comment by naddison36** [comment]
This will be fixed in the next 2.4.4 release

**Comment by naddison36** [comment]
This has been included in Release v2.5.0
